diagnostic.md

```diff
@@ -8,6 +8,11 @@ Assume all code executes in strict mode.
 ## Question 1
 
 Please list the JavaScript primitives and give one example of each.
+Number: 5
+String: "This is a string!"
+Boolean: true
+Object: {key: value}
+Array: ["an", "array", "of", "strings"]
```

Comment:
Arrays and Objects are not actually primitives. Please take a look at the JS Basics lesson again: https://github.com/ga-wdi-boston/js-basics